app/addons/cors/components.react.jsx

```diff
@@ -225,7 +225,8 @@ define([
         origins: corsStore.getOrigins(),
         isAllOrigins: corsStore.isAllOrigins(),
         configChanged: corsStore.hasConfigChanged(),
-        savingStatus: corsStore.getSavingStatus()
+        shouldSaveChange: corsStore.shouldSaveChange()
+        //savingStatus: corsStore.getSavingStatus()
```

Comment:
if not we can also delete https://github.com/apache/couchdb-fauxton/blob/9343084ace991386fab014529efdfdc67e10f389/app/addons/cors/stores.js#L35 et al (setter, actiontype...)